code/crates/engine/src/consensus.rs

```diff
@@ -1142,9 +1178,19 @@ where
         state: &mut State<Ctx>,
     ) -> Result<(), ActorProcessingErr> {
         info!("Stopping...");
-
         state.timers.cancel_all();
-
         Ok(())
     }
 }
+
+fn should_buffer<Ctx: Context>(msg: &Msg<Ctx>) -> bool {
+    !matches!(
+        msg,
+        Msg::StartHeight(..)
+            | Msg::TimeoutElapsed(..)
```

Comment:
Why do we need to let the `TimeoutElapsed` through?

Reply:
No that reason I can think of actually, since no timeouts are even scheduled by that point. I will remove it.